### PR TITLE
Setup package for hex.pm

### PR DESCRIPTION
**Pull request by mager**
Following instructions from https://hex.pm/docs/publish

### CODE REVIEW

**Comment by mager** [comment]
r?

File: mix.exs [hunk 1]
```diff
@@ -8,7 +8,9 @@ defmodule ConsulMutEx.Mixfile do
       elixir: "~> 1.4",
       build_embedded: Mix.env == :prod,
       start_permanent: Mix.env == :prod,
+      description: description(),
       deps: deps(),
+      package: package(),
```
Comment (wezleytsai) [inline mix.exs]: why can't you just define these inline instead of making functions?

Comment (mager) [inline mix.exs]: Just following the example from hex.pm/docs/publish

Comment (mager) [inline mix.exs]: Functional programming? :)

**Pull request merged**